frontend/src/components/Pages/MissionHistoryPage/HistoricMissionCard.tsx

```diff
@@ -51,6 +58,9 @@ export const HistoricMissionCard = ({ index, mission }: IndexedMissionProps) =>
             <Table.Cell>
                 <MissionEndTimeDisplay mission={mission} />
             </Table.Cell>
+            <Table.Cell>
+                <MissionRestartButton mission={mission} hasFailedTasks={missionHasFailedTasks} />
```

Comment:
Consider using AddOutlined as icon instead of Replay as that is what we have in figma